Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/client/helper.go

```diff
@@ -261,6 +263,18 @@ func TransportFor(config *Config) (http.RoundTripper, error) {
 			transport = http.DefaultTransport
 		}
 	}
+
+	switch {
```

Comment:
> I would guard this with a package var that defaults to false to prevent turning it on server-side

@smarterclayton permission to create a global?